bindings/rust/evmc-vm/src/lib.rs

```diff
@@ -61,6 +67,219 @@ impl ExecutionResult {
     }
 }
 
+impl<'a> ExecutionContext<'a> {
+    pub fn new(_message: &'a ffi::evmc_message, _context: &'a mut ffi::evmc_context) -> Self {
+        let _tx_context =  unsafe {
+            assert!(!(*(_context.host)).get_tx_context.is_some());
+            (*(_context.host)).get_tx_context.unwrap()(_context as *mut ffi::evmc_context)
+        };
+
+        ExecutionContext {
+            message: _message,
+            context: _context,
+            tx_context: _tx_context,
+        }
+    }
+
+    pub fn call_kind(&self) -> &ffi::evmc_call_kind {
+        &self.message.kind
+    }
+
+    pub fn is_static(&self) -> bool {
+        self.message.flags == 1
```

Comment:
Not sure we want this helper, but in any case this should use `ffi::EVMC_STATIC` or `ffii::evmc_flags::EVMC_STATIC`.